py_gtfs_rt_ingestion/lib/s3_utils.py

```diff
@@ -80,13 +81,6 @@ def _move_s3_object(filename: str, to_bucket: str) -> None:
     :param filename - expected as 's3://my_bucket/the/path/to/the/file.json
     :param to_bucket bucket name
     """
-    process_logger = ProcessLogger(
-        "move_s3_object",
-        filename=filename,
-        to_bucket=to_bucket,
-    )
-    process_logger.log_start()
-
```

Comment:
remove individual object move logging, was spamming splunk output

function now returns `1` if move was success, `0` if failure, fail count given in `move_s3_objects` log